pyproject.toml

```diff
@@ -1,6 +1,45 @@
-[build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+[tool.poetry]
+name = "momento"
+version = "0.0.0"
+
+authors = ["Momento <open-source@momentohq.com>"]
```

Comment:
@malandis can you take point on sync'ing w/ @ksmotiv8 and making sure we have this alias actually made before we merge this pr please.